### ISSUE TITLE
Plant Disease Prediction

### ISSUE THREAD

**Issue by its-kumar-yash** [Niketkumardheeryan/ML-CaPsule#527]
Overview:
Aim is to develop a predictive model for Plant disease using TensorFlow.

Objective:
The primary objective of this project is to create a machine-learning model that can accurately predict the disease by analyzing Plant's leaf Image.

I would like this issue to be assigned to me as part of GSSoC'24. Thank you.

**Comment by Devanik21** [comment]
Enthusiastic to take on this challenge! Please assign the task to me for GSSOC contribution.

**Comment by Dhananjaynagar** [comment]
I am willing to work on this. 

I will be making and implementing the Use of------
**CNNs **

Convolutional neural network (CNN) is a type of artificial neural network used primarily for image recognition and processing. 
 
Hefty amount of model training with data will be implemented. I will also try to deploy this as an application using **Flutter ** 

Here is the flow chart![images.png](https://github.com/Niketkumardheeryan/ML-CaPsule/assets/110812681/e27772e1-fc84-4b25-a94a-bba728dde79c)

So @Niketkumardheeryan please assign me this for my GSSOC, 24

**Comment by kyra-09** [comment]
Hey, @Niketkumardheeryan 

Participation role :- Contributor/GSSOC-2024

Approach to this project :- 
1.) I will use Keras API and different pre-trained models such as VGG , MobileNet , InceptionV3 and more with higher accuracy for detection in disease in plants using the dataset.
2.) dataset I will use :- https://www.kaggle.com/datasets/rashikrahmanpritom/plant-disease-recognition-dataset.
3.) analyse and make a requirement.txt and ReadMe file for it.

**Kindly assign this issue to me**

**Comment by invigorzz313** [comment]
The issue has been assigned on FCFS basis.
@its-kumar-yash Do follow the project guidelines for PR.
